README.md

```diff
@@ -74,7 +74,18 @@ $version = XdebugHandler::getSkippedVersion();
 ```
 
 ### Output
-The library is silent unless the `-vvv` command-line option is present, which results in the output of simple status messages.
+The `setLogger` method enables the output of status messages to an external PSR3 logger, with either `DEBUG` or `WARNING` log levels. Alternatively, a CLI logger is available that writes these messages to `stdout` when running in a CLI process.
+
+```php
+use Composer\XdebugHandler\XdebugHandler;
+
+$xdebug = new XdebugHandler('myapp');
+// Provide a PSR3 logger
+$xdebug->setLogger($myLogger);
+
+// Or use the CLI logger if appropriate
+$xdebug->setLogger(new Composer\XdebugHandler\CliLogger);
```

Comment:
Can we just give an example with [monolog](https://packagist.org/packages/monolog/monolog)? or An anonymous class?

Reply:
Like
```php
$xdebug->setLogger(new Monolog\Handler\StreamHandler(STDERR))
```
and all further questions about the implementation and needs a developer might have don't need to be answered in this repo.

Reply:
I think I prefer a vague `$myLogger` (implementation specific), rather than an explicit example of one.

Reply:
I'm fine with that too :)